types/lolex/index.d.ts

```diff
@@ -167,23 +174,42 @@ export interface LolexClock<TTimerId extends TimerId> {
  */
 export declare function createClock<TClock extends Clock>(now?: number | Date, loopLimit?: number): TClock;
 
-/**
- * Creates a clock and installs it globally.
- * 
- * @param now   Current time for the clock, as with lolex.createClock().
- * @param toFake   Names of methods that should be faked.
- * @type TClock   Type of clock to create.
- * @usage lolex.install(["setTimeout", "clearTimeout"]);
- */
-export declare function install<TClock extends Clock>(now?: number | Date, toFake?: FakeMethod[]): TClock;
+
+export interface LolexInstallOpts {
+    /**
+     * installs lolex onto the specified target context (default: global)
+     */
+    target?: any;
```

Comment:
Nit: the existing definitions keep a blank line between each member and the following `/**` comment. It would be cleaner to keep that existing style.

Reply:
fixed